src/camera/camera3d.jl

```diff
@@ -249,10 +249,17 @@ function add_translation!(scene, cam::Camera3D)
             elseif event.action == Mouse.release && dragging[]
                 mousepos = mouseposition_px(scene)
                 dragging[] = false
-                diff = (last_mousepos[] - mousepos) * 0.01f0 * translationspeed[]
+                diff = (last_mousepos[] - mousepos) 
                 last_mousepos[] = mousepos
-                viewdir = cam.lookat[] - cam.eyeposition[]
-                translate_cam!(scene, cam, cam.zoom_mult[] * norm(viewdir) * Vec3f0(diff[1], diff[2], 0f0))
+                if cam.attributes[:projectiontype][] == Orthographic
+                    aspect = Float32((/)(widths(scene.px_area[])...))
+                    aspect_scale = Vec2f0(1f0 + aspect, 1f0 + 1f0 / aspect)
+                    diff = cam.zoom_mult[] * diff .* aspect_scale ./ widths(scene.px_area[])
+                else
+                    viewdir = cam.lookat[] - cam.eyeposition[]
+                    diff = 0.002f0 * cam.zoom_mult[] * norm(viewdir) * diff
```

Comment:
Since this is also used in `on(camera(scene), scene.events.mouseposition) do mp` below, you could write a closure for computing `diff_orthographic` or `diff_perpective` to avoid duplicate code ...